doc/howtos/image-configuration.rst

```diff
@@ -0,0 +1,100 @@
+Configuring Refkit images
```

Comment:
That's right. The intended audience is anyone who needs to build a customized Refkit image.

I think there would be a need for some general purpose document, which would be called "How to create a custom IoT distribution based on Refkit". That should include things like:
1. How to create an own layer for Refkit customization? Is it better to do a layer inside Refkit or add Refkit inside some other layer?
2. How to rebase when refkit changes? How to solve typical errors in the process?
3. How to properly do builds and push resulting updates to target devices?
4. How to develop and maintain your own recipes on top of Refkit?
And finally:
5. How to configure packages in Refkit? (this document)

I'm not sure if extending this PR would be the right place to do that. Instead We could do a set of documents answering these questions and link them together on top level.